lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivationSpec.hs

```diff
@@ -577,3 +623,17 @@ genLegacyAddress range = do
 
 instance Arbitrary SomeMnemonic where
     arbitrary = SomeMnemonic <$> genMnemonic @12
+
+-- | Encrypt password using Scrypt function with the following parameters:
+-- logN = 14
+-- r = 8
+-- p = 1
+-- These parameters are in Scrypt.defaultParams
+encryptPasswordWithScrypt
+    :: Passphrase "raw"
+    -> IO (Hash "encryption")
+encryptPasswordWithScrypt p =
+    Hash . Scrypt.getEncryptedPass <$>
+    Scrypt.encryptPassIO Scrypt.defaultParams (Scrypt.Pass $ BA.convert passwd)
+  where
+    (Passphrase passwd) = preparePassphrase EncryptWithScrypt p
```

Comment:
side note, `Passphrase` has an instance of `ByteArrayAccess`, to `BA.convert` can be called on it directly, no need to pattern match on this output :+1:

Reply:
`Passphrase` is `BA.ScrubbedBytes` inside and `Scrypt.Pass` requires `B8.ByteString` - hence I used `BA.convert` here